src/commands/logs.rs

```diff
@@ -0,0 +1,168 @@
+use std::ops::Sub;
+use std::time::Duration;
+
+use anyhow::{anyhow, Result};
+use chrono::Utc;
+use cloud_openapi::models::Entry;
+
+use cloud::{client::Client as CloudClient, CloudClientExt, CloudClientInterface};
+
+use crate::commands::create_cloud_client;
+use crate::commands::deploy::SPIN_DEPLOY_CHANNEL_NAME;
+use crate::opts::*;
+use clap::Parser;
+use uuid::Uuid;
+
+/// Fetch and tail logs of an application deployed to the Fermyon Cloud.
+#[derive(Parser, Debug)]
+#[clap(about = "fetch logs for an app from Fermyon Cloud")]
+pub struct LogsCommand {
+    /// Find app to fetch logs for in the Fermyon instance saved under the specified name.
+    /// If omitted, Spin looks for app in default unnamed instance.
+    #[clap(
+        name = "environment-name",
+        long = "environment-name",
+        env = DEPLOYMENT_ENV_NAME_ENV
+    )]
+    pub deployment_env_id: Option<String>,
+
+    /// App name
+    #[clap(name = "app-name", long = "app-name")]
+    pub app_name: String,
+
+    /// Follow logs output
+    #[clap(name = "follow", long = "follow")]
+    pub follow: bool,
+
+    /// Number of lines to show from the end of the logs
+    #[clap(name = "tail", long = "tail", default_value = "10")]
+    pub max_lines: i32,
+
+    /// interval in secs to refresh logs from cloud
```

Comment:
Consistent capitalisation
```suggestion
    /// Interval in secs to refresh logs from cloud
```